internal/edgeapi/backend/k8s/backend.go

```diff
@@ -176,6 +177,27 @@ func (b *backend) UpdateStatus(ctx context.Context, name, namespace string, hear
 	return b.heartbeatHandler.Process(ctx, name, namespace, heartbeat)
 }
 
+func (b *backend) GetPlaybookExecutions(ctx context.Context, deviceID, namespace string) ([]*models.PlaybookExecution, error) {
+	logger := b.logger.With("DeviceID", deviceID, "Namespace", namespace)
+	response := []*models.PlaybookExecution{}
+	edgeDevice, err := b.repository.GetEdgeDevice(ctx, deviceID, namespace)
+	if err != nil {
+		return nil, err
+	}
+
+	for labelName, labelValue := range edgeDevice.Labels {
+		if labels.IsEdgeConfigLabel(labelName) {
+			playbookExecution, err := b.repository.GetPlaybookExecution(ctx, labelValue, namespace)
+			if err != nil {
+				logger.Error(err, "cannot get playbook execution", "playbook execution name", labelValue, "namespace", namespace)
```

Comment:
the namespace can be removed from this log since it is already set for the logger in line 181